### PR TITLE
Version Number in the Frmmain code file

### PR DESCRIPTION
**Pull request by derekagorhom**
From comments on PR #9159
https://github.com/IDEMSInternational/R-Instat/pull/9159#discussion_r1780639924
I have added the version number function into the code file 
@N-thony , @Patowhiz can you check and confirm
Thanks

### CODE REVIEW

File: instat/frmMain.vb [hunk 1]
```diff
@@ -585,6 +585,12 @@ Public Class frmMain
         mnuTbLan.Visible = bVisible
     End Sub
 
+    Public Sub VersionNumber()
```
Comment (Patowhiz) [inline instat/frmMain.vb:588]:
```suggestion
    Public Sub SetAppVersionNumber()
```

**Review by Patowhiz — CHANGES_REQUESTED** [review]
@derekagorhom looks good. I have just made 1 suggestion change on the name of the subroutine to be more descriptive.

File: instat/frmMain.vb [hunk 2]
```diff
@@ -225,7 +225,7 @@ Public Class frmMain
         '--------------------------------------
         CreateAdditionalLibraryDirectory()
         '-------------------------------------
-
+        VersionNumber()
```
Comment (Patowhiz) [inline instat/frmMain.vb:228]:
```suggestion
        SetAppVersionNumber()
```

**Review by Patowhiz — DISMISSED** [review]

**Review by Patowhiz — APPROVED** [review]
@N-thony this now looks good. Kindly check and merge.

**Review by rdstern — APPROVED** [review]
@N-thony I don't have #9163 though I see it here.  I am approving this one blinfly

**Comment by rdstern** [comment]
@Patowhiz and @N-thony is this still needed?  The version number seems to be there now.  Should this be merged or closed?

**Comment by Patowhiz** [comment]
@rdstern this is meant to put the code for displaying the version number in the correct place. 
@derekagorhom the code looks good and ready for merging. I was wondering whether we should remove the one in the designer file? Then we can merge this.

**Comment by derekagorhom** [comment]
> @rdstern this is meant to put the code for displaying the version number in the correct place. @derekagorhom the code looks good and ready for merging. I was wondering whether we should remove the one in the designer file? Then we can merge this.

@Patowhiz I have removed the code in the designer file.....can you confirm and approve again.
Thanks

**Review by rdstern — APPROVED** [review]
@Patowhiz sounds good, so I am approving.  I don't know if you would simply like to approve and merge, or get @N-thony to do a final check and merge?

**Review by Patowhiz — APPROVED** [review]

**Pull request merged**